docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,619 @@
+                values:
+                - backend
+            topologyKey: kubernetes.io/hostname
+      containers:
+      - image: quay.io/openshift/origin-cli:4.8
+        imagePullPolicy: IfNotPresent
+        name: oc-rsync
+        command:
+          - /bin/bash
+          - -c
+          - |
+            #!/bin/bash
+            oc \
+            --server=$K8S_API \
+            --token=$K8S_TOKEN \
+            --namespace=$K8S_NAMESPACE \
+            rsync \
+            --delete=true \
+            --watch=true \
+            /rsync-source/ \
+            "$(oc --server=$K8S_API --token=$K8S_TOKEN --namespace=$K8S_NAMESPACE get pod -l app.kubernetes.io/name=rsync-destination -o jsonpath={.items[0].metadata.name}):/rsync-destination/"
+        env:
+        - name: K8S_API
+          value: https://<kubernetes-api>:6443
+        - name: K8S_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: rsync-destination-oc-token
+              key: token
+        - name: K8S_NAMESPACE
+          value: rsync-test
+        volumeMounts:
+        - mountPath: /rsync-source
+          name: rsync-source
+      volumes:
+      - name: rsync-source
+        persistentVolumeClaim:
+          claimName: rsync-source
+--
+<1> Please refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/[the Kubernetes documentation] on common labels.
+
+
+IMPORTANT: Be aware that `oc rsync` has different options than `rsync` itself.
+
+[source,bash]
+--
+Options:
+     --compress=false: compress file data during the transfer
+ -c, --container='': Container within the pod
+     --delete=false: If true, delete files not present in source
+     --exclude=[]: When specified, exclude files matching pattern
+     --include=[]: When specified, include files matching pattern
+     --no-perms=false: If true, do not transfer permissions
+     --progress=false: If true, show progress during transfer
+ -q, --quiet=false: Suppress non-error messages
+     --strategy='': Specify which strategy to use for copy: rsync, rsync-daemon, or tar
+ -w, --watch=false: Watch directory for changes and resync automatically
+--
+
+=== File Integrity Check
+
+After the migration, you should check the integrity of your data with the following commands:
+
+.Calculate the checksum for all files at the origin
+[source,bash]
+--
+$ find . -type d -exec sh -c "cd '{}' && find . -maxdepth 1 -type f ! -name COPYSHA1SUMS -printf '%P\0' | xargs -r0 sha1sum -- > COPYSHA1SUMS" \;
+--
+
+.Create a log with a verification of all files at the destination
+[source,bash]
+--
+$ cd <path> && find . -type d -exec sh -c "cd '{}' && echo '{}' && sha1sum -c COPYSHA1SUMS" \; > sha1sums-verify-log-$(date +%F-%H-%M).log 2>&1
+--
+
+== Container Images
+
+Since APPUiO Cloud is based on OpenShift 4, there are new requirements for your container images. This section contains all the required steps for adapting your images to the new environment.
```

Comment:
```suggestion
Since APPUiO Cloud is based on OpenShift 4, there are new requirements for your container images.
This section contains all the required steps for adapting your images to the new environment.
```